### PR TITLE
Hvu dataset

### PR DESCRIPTION
**Pull request by NikhilBartwal**
# Add Dataset

* Holistic Video Understanding dataset
* Issue Reference: <https://github.com/tensorflow/datasets/issues/2175>
* `dataset_info.json` Gist: <https://gist.github.com/NikhilBartwal/95e74e25979162df66d2fe7ae7355f0a>
  
## Description
Action recognition has been advanced in recent years by benchmarks with rich annotations.
However, research is still mainly limited to human action or sports recognition - focusing on a highly specific video understanding task and thus leaving a significant gap towards describing the overall content of a video. We fill in this gap by presenting a large-scale "Holistic Video Understanding Dataset"~(HVU).

HVU is organized hierarchically in a semantic taxonomy that focuses on multi-label and multi-task video understanding as a comprehensive problem that encompasses the recognition of multiple semantic aspects in the dynamic scene.
HVU contains approx.~577k videos in total with ~13M annotations for training and validation set spanning over ~3k classes.
HVU encompasses semantic aspects defined on categories of scenes, objects, actions, events, attributes and concepts, which naturally captures the real-world scenarios.

The added dataset consists of 4 different `BuilderConfig`s , two sample datasets and two complete datasets of 64x64 and 128x128 resolutions respectively. Each `BuilderConfig` specifies a train and validation split with the sample dataset consisting of 100 train and validation files each.

The dataset was generated with the help of Apache Beam and therefore, due to resource limitations, the `dataset_info.json` file linked here specifies the 64x64 sample dataset only.

<description>
  
## Checklist
* [x] Address all TODO's
* [x] Add alphabetized import to subdirectory's `__init__.py`
* [x] Run `download_and_prepare` successfully
* [x] Add [checksums file](https://www.tensorflow.org/datasets/add_dataset#2_run_download_and_prepare_locally)
* [x] Properly cite in `BibTeX` format
* [x] Add passing test(s)
* [x] Add test data
* [x] If using additional dependencies (e.g. `scipy`), use [lazy_imports](https://www.tensorflow.org/datasets/add_dataset#extra_dependencies) (if applicable)
* [x] Add data generation script (if applicable)
* [x] [Lint](https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style) code

### CODE REVIEW

**Comment by NikhilBartwal** [comment]
@bhack @Conchylicultor Could you have a look at the PR? Thanks!

**Comment by bhack** [comment]
Sorry I still don't have the bandwidth to review this. Can we have a first pass from a TFDS maintainer?
